app/components/partner/credit-transfer/index.scss

```diff
@@ -0,0 +1,121 @@
+.credit-transfer {
+  margin: 0 0.35rem;
+}
+
+.credit-transfer-btn {
+  border-radius: 50%;
+}
+
+.client {
+  display: grid;
+  grid-template-columns: auto auto;
+  align-items: center;
+  column-gap: 1rem;
+  justify-content: center;
+}
+
+.invalid-credits {
+  color: $red;
+  font-size: 0.8rem;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  justify-self: center;
+}
+
+.title {
+  text-transform: capitalize;
+  font-weight: bold;
+  color: rgba(#000, 0.7);
+  max-height: 92%;
+  font-size: 1.1rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  &.empty-title {
+    color: #424651;
+    font-size: 1em;
+    font-style: italic;
+  }
+}
+
+.credit-tranfer {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  row-gap: 1.5rem;
+}
+
+.transferable-credits {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #0063b2ff;
+  text-align: center;
+}
+
+.tranfer-count {
+  display: flex;
+  align-items: center;
+}
+
+.input-count {
+  width: 8rem;
+  height: 2rem;
+  border: none;
+  border-bottom: 1px solid;
+  color: #0063b2ff;
+  font-size: 1.5rem;
+  text-align: center;
+  font-weight: bold;
+}
+
+.credit-type {
+  font-size: 0.95em;
+  color: #0063b2ff;
+  width: 3em;
+}
+
+.tranfer-btn {
+  justify-self: center;
+}
+
+.bal-credits {
+  text-transform: capitalize;
+  font-size: 0.95rem;
+}
+
+.disabled-btn {
+  opacity: 0.5;
+  cursor: default;
+}
+
+.row {
+  display: flex;
+  padding: 0.2rem 0;
+}
+
+.key {
+  width: 30%;
+  font-size: 0.85rem;
+  display: flex;
+  align-items: center;
+}
+
+.value {
+  width: 70%;
+  display: flex;
+  align-items: center;
+}
+
+.action-btns {
+  display: flex;
+  justify-content: center;
+}
+
+.back-btn {
+  border-color: rgba(#424651, 0.7);
+  color: rgba(#424651, 0.7);
+  &:hover {
+    border-color: rgba(#424651, 1);
+    color: rgba(#424651, 1);
+  }
+}
```

Comment:
Don't introduce new buttons, use the existing ones

Reply:
Yeah, saw the is-default class and used the same for back button